utility/confirmation.go

```diff
@@ -87,3 +87,19 @@ func UserConfirmedOverwrite(resourceType string, ignoringConfirmed bool) bool {
 
 	return true
 }
+
+// ReadUserInputAndCheckIfYes is a function that can retrieve user input in form of string and checks if it is a yes.
+func ReadUserInputAndCheckIfYes(in io.Reader) error {
```

Comment:
I would create a more reusable function like: 
`UserAccepts(in io.Reader) (bool, error)`
possible answers: true (YES), false(NO), error